tests/unit/lib/Any_unittest.cc

```diff
@@ -691,3 +691,69 @@ TEST(Any, lazy_cloneable_visitable)
 	EXPECT_FALSE(any.visitable());
 	EXPECT_THROW(any.visit(nullptr), std::logic_error);
 }
+
+TEST(AnyParameterProperties, old_api_default)
+{
+	AnyParameterProperties params = AnyParameterProperties();
+
+	EXPECT_EQ(params.get_description(), "No description given");
+	EXPECT_EQ(
+	    params.get_model_selection(),
+	    EModelSelectionAvailability::MS_NOT_AVAILABLE);
+	EXPECT_EQ(
+	    params.get_gradient(), EGradientAvailability::GRADIENT_NOT_AVAILABLE);
+	EXPECT_FALSE(params.get_model());
+}
+
+TEST(AnyParameterProperties, new_api_default)
+{
+	AnyParameterProperties params = AnyParameterProperties();
+
+	EXPECT_EQ(params.get_description(), "No description given");
```

Comment:
not sure whether it is good to test this .. imagine we want to change it -- need to change more than one test.

Reply:
True!